File: Objective-C/Reference App/9. Insert Map View (with Storyboard)/InsertMapViewWithStoryboardViewController.m
//
//  InsertMapViewWithStoryboardViewController.m
//  Reference App
//
//  Created by David Deller on 5/3/18.
//  Copyright © 2018 TallyGo. All rights reserved.
//

#import "InsertMapViewWithStoryboardViewController.h"

@interface InsertMapViewWithStoryboardViewController ()

@property (nonatomic, weak) IBOutlet TGMapView *mapView;

@end

@implementation InsertMapViewWithStoryboardViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    // Part 2: Customize it to your liking
    self.mapView.tintColor = UIColor.greenColor;
    self.mapView.zoomLevel = 12;
    self.mapView.showsUserLocation = YES;
    [self.mapView setUserTrackingMode:MGLUserTrackingModeFollow animated:NO];
}

@end

File: Swift 4/Reference App/9. Insert Map View (with Storyboard)/InsertMapViewWithStoryboardViewController.swift
//
//  InsertMapViewWithStoryboardViewController.swift
//  Reference App
//
//  Created by David Deller on 5/1/18.
//  Copyright © 2018 TallyGo. All rights reserved.
//

import UIKit
import TallyGoKit

class InsertMapViewWithStoryboardViewController: ExampleViewController {

    @IBOutlet weak var mapView: TGMapView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        // Part 2: Customize it to your liking
        mapView.tintColor = UIColor.green
        mapView.zoomLevel = 12
        mapView.showsUserLocation = true
        mapView.setUserTrackingMode(.follow, animated: false)
    }

}
